### DISCUSSION TITLE
How do I change the default confirm button?

### DISCUSSION THREAD

**Discussion by mak448a** [dialogic-godot/dialogic#2269] [Q&A] [answered]
I'd like to change it from space to control.

**Comment by Jowan-Spooner** [comment, marked as answer]
All input things are defined in godot's InputMap (in the project settings).
For dialogic advance it's using the dialogic_default_action by default.
Choices are buttons though and they use the ui_confirm action (a built-in godot action). Be careful when editing those, as they affect all buttons/control nodes in the project.
